Update readmes (#59)
Sindri-Labs/sindri-resources: circuit_tutorials/gnark/README.md

# Gnark Circuit Tutorials

> In this directory, we provide a handful of Gnark circuits tutorials which have been prepared to be used with Sindri.

Reviewer: ```suggestion
In this directory, we provide a handful of Gnark circuit tutorials which have been prepared for use with Sindri.
```
